### ISSUE TITLE
W5 AT: show asset thumbnail in asset manager

### ISSUE THREAD

**Issue by hirokiterashima** [WISE-Community/WISE#602]
Request by Lauren:
When the author clicks on an image asset, show the thumbnail version of it.

**Comment by breity** [comment]
Yep, I'm planning to have a more full-featured asset manager for the authoring tool that will show thumbnails (and full-size previews on selection) of both images and videos.

**Comment by hirokiterashima** [comment]
@breity ok, we'll create another issue when we start working on the full-featured asset manager. 

For now I show a simple preview popup when the author clicks on the "view" button for an image asset. Very simple and it'll be enough for our two authors.
![screen shot 2016-04-11 at 2 41 10 pm](https://cloud.githubusercontent.com/assets/119416/14444634/9dd494f2-fffb-11e5-8d79-22123dcec234.png)

One thing I realized is that we currently don't have a way to know the mime type of each file. This will be useful for knowing what to do with the asset (show it in a video player, img tag, etc...) I created a ticket here: #609
